frontend/test/metabase/scenarios/native/native-mongo.cy.spec.js

```diff
@@ -24,13 +25,19 @@ describe("scenarios > question > native > mongo", () => {
       parseSpecialCharSequences: false,
     });
     cy.get(".NativeQueryEditor .Icon-play").click();
-    cy.findByText("18,760");
+
+    cy.wait("@dataset");
+
+    cy.findByTextEnsureVisible("18,760");
 
     cy.findByText("Save").click();
 
+    cy.findByTextEnsureVisible("Save question");
+
     modal().within(() => {
       cy.findByLabelText("Name")
-        .focus()
+        .clear()
+        .should("have.value", "")
```

Comment:
Somehow this construct has a better chance of success, then simply `.focus()`. Not entirely sure why.

Reply:
Perhaps #18150 is at play

Reply:
Good point @flamber! I think it's exactly that issue.